msbuild/Xamarin.Shared/Xamarin.Shared.props

```diff
@@ -226,6 +226,19 @@ Copyright (C) 2020 Microsoft. All rights reserved.
 		<_NativeExecutableName Condition="'$(_NativeExecutableName)' == ''">$(AssemblyName)</_NativeExecutableName>
 	</PropertyGroup>
 
+	<PropertyGroup Condition="'$(EnableDiagnostics)' == 'true' And '$(DiagnosticConfiguration)' == ''">
```

Comment:
Would setting any of the Diagnostic* properties implicitly mean EnableDiagnostics?

Reply:
The latest changes do this, looks like it matches dotnet/android's logic.